code/queryHandlers/RESTfulAPI_DefaultQueryHandler.php

```diff
@@ -348,9 +348,6 @@ function createModel($model, SS_HTTPRequest $request)
       );
     }
 
-    $newModel = Injector::inst()->create($model);
-    $newModel->write();
-
     return $this->updateModel($model, $newModel->ID, $request);
```

Comment:
What a bummer, I've commited on the wrong branch.

That line should actually be

    return $this->updateModel($model, 0, $request);